src/Microsoft.ML.Data/Transforms/ExtensionsCatalog.cs

```diff
@@ -90,4 +91,14 @@ public static ColumnSelectingEstimator DropColumns(this TransformsCatalog catalo
             => new ColumnSelectingEstimator(CatalogUtils.GetEnvironment(catalog),
                 keepColumns, dropColumns, keepHidden, ignoreMissing);
     }
+
+    public static class ValueMappingCatalog
```

Comment:
>ValueMappingCatalog [](start = 24, length = 19)

I think it's more suitable for CoversionCatalog. or what the name of place with TermTransform